### DISCUSSION TITLE
Display value for Linked records [May 13]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5691] [Discord Threads]
> 🙋‍♂️ **Alexey Zaytsev**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### https://docs.nocodb.com/setup-and-usages/display-value#how-to-set-display-value-  
#  
> 🙋‍♂️ **Alexey Zaytsev**  
#### Thank you!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1096052089957916673)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
